src/Bicep.LangServer/Server.cs

```diff
@@ -42,6 +43,7 @@ public Server(Stream input, Stream output, CreationOptions creationOptions)
         
         private Server(CreationOptions creationOptions, Action<LanguageServerOptions> onOptionsFunc)
         {
+            BicepDeploymentsInterop.Initialize();
```

Comment:
There's a call to initialize this, but no corresponding project changes to add a project ref?

Reply:
It's something I realized that I missed when adding the dependency on the Azure.Deployments library, so not strictly related to this PR. The intention with this library is that it should always be initialized before being used, or it may fail in an unexpected manner. This was a deliberate design decision at the time, so that we'd fail fast, rather than use the library in an unsupported manner.

Reply:
Also, to note - we should have an internal discussion about how to handle this going forwards. Requiring initialization doesn't feel optimal as it is easy to overlook. Not requiring initialization has some important drawbacks worth discussing.